### PR TITLE
update sponsors block

### LINKED ISSUE #117: Update sponsor links/images
@markstory 

We still have the old sponsors for api docs footer.

image

### PR DESCRIPTION
**Pull request by Arhell**
related https://github.com/cakephp/cakephp-api-docs/issues/117

### CODE REVIEW

File: templates/footer.twig [hunk 1]
```diff
@@ -51,11 +51,18 @@
...
 						</a>
-						<div class="mt10">
-							<a href="https://www.rackspace.com/">
-								<img alt="Rackspace" src="./resources/rackspace.png">
-							</a>
-						</div>
+                        <div class="mt10">
+                            <a href="https://pingping.io">
+                                {var logoImg = 'resources/pingping.png'}
+                                <img alt="Rackspace" src="{$logoImg|staticFile}">
```
Comment (othercorey) [inline templates/footer.twig]: This should just be the html. 2.x was re-written and it not a .latte file.

Comment (Arhell) [reply templates/footer.twig]: Unfortunately I didn't find .html here .twig

Comment (othercorey) [reply templates/footer.twig]: Yes. The twig has just html. Just use `src="./resources/pingping.png"`.

Comment (Arhell) [reply templates/footer.twig]: Done, thanks.

**Pull request merged**